### ISSUE TITLE
Container Storage Interface support

### ISSUE THREAD

**Issue by Logibox** [hpe-storage/python-hpedockerplugin#462]
The [container storage interface][2] (CSI) is now GA in Kubernetes 1.13 ([see here][2])

CSI is a more generic way of presenting storage to container orchestration systems that is not docker dependant (for example I don't think you can use this plugin with CRI-O atm, unless i'm mistaken).

Are there any plans to implement this interface for 3PAR?

[1]: https://github.com/container-storage-interface/spec
[2]: https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/

**Comment by prablr79** [comment]
@Logibox Yes there are plans to implement CSI for 3PAR, however dates not worked out with Product Marketing team.

**Comment by cdmaxson1** [comment]
We are also very interested in a CSI version of the plugin.

**Comment by prablr79** [comment]
@cdmaxson1 HPE 3PAR  roadmap is on CSI driver development, engineering sure will bring up in during Next review meeting with HPE Product Mgmt .

**Comment by prablr79** [comment]
@cdmaxson1 Please also send out email to Srik on this regard.

**Comment by cdmaxson1** [comment]
Will do.

**Comment by axelheindrichs** [comment]
Any news on this? ETA known?

**Comment by amitk1977** [comment]
CSI support with 3PAR is under development and ETA is 1H2020

**Comment by axelheindrichs** [comment]
Hi there, do you have an update on the progress or ETA? 
Would it be possible to early access to the driver?
Is this the one to watch? https://github.com/hpe-storage/csi-driver
Cheers
